app/src/main/java/com/github/se/project/ui/navigation/NavigationActions.kt

```diff
@@ -34,6 +34,7 @@ object Screen {
   const val LESSONS_REQUESTED = "Lessons Requested Screen"
   const val TUTOR_LESSON_RESPONSE = "Tutor Lesson Response Screen"
   const val SEARCH = "Search Screen" // Find tutor / find student screen
+  const val MAP_LOC_PICKER = "Map Screen"
```

Comment:
this screen constant is not used since the map picker you implemented is just a composable. Maybe you left it for future modification, but consider removing it in the other case :)

Reply:
Good point I am removing it